client/src/components/displayCard.js

```diff
@@ -0,0 +1,32 @@
+//import { urlencoded } from "express";
+import { useState } from "react";
+
+const DisplayCard = ({ card }) => {
+  //QUALITY OF LIFE CODE
+  // function will make special char appear on page, will pass them through function as string
+  function decodeHtml(html) {
+    var txt = document.createElement("textarea");
+    txt.innerHTML = html;
+    return txt.value;
+  }
+  console.log(card, "this is one card");
+
+  return (
+    <div className="display-card">
+      <h2>{JSON.stringify(card.name)}</h2>
+      <p>
+        <b>Mana Cost:</b> {JSON.stringify(card.manaCost)}
```

Comment:
if manaCost and originalText are strings, then the `JSON.stringify` can be removed (this function is mainly useful for string-ifying things like arrays and objects) - other data types can be rendered in react as is